api/tests/shared/integration/infrastructure/repositories/assessment-result-repository_test.js

```diff
@@ -518,4 +518,70 @@ describe('Integration | Repository | AssessmentResult', function () {
       });
     });
   });
+
+  describe('#updateToAcquiredLowerLevelComplementaryCertification', function () {
+    context('when the assessment result exists', function () {
+      it('should update the assessment result', async function () {
+        // given
+        databaseBuilder.factory.buildCertificationCourse({ id: 1 });
+        databaseBuilder.factory.buildUser({ id: 100 });
+        databaseBuilder.factory.buildAssessment({ id: 2, certificationCourseId: 1 });
+        const competenceMark1 = domainBuilder.buildCompetenceMark({
+          id: 200,
+          level: 3,
+          score: 33,
+          area_code: 'area1',
+          competence_code: 'compCode1',
+          competenceId: 'recComp1',
+          assessmentResultId: 4,
+        });
+        const competenceMark2 = domainBuilder.buildCompetenceMark({
+          id: 201,
+          level: 1,
+          score: 2,
+          area_code: 'area2',
+          competence_code: 'compCode2',
+          competenceId: 'recComp2',
+          assessmentResultId: 4,
+        });
+        const assessmentResult = domainBuilder.buildAssessmentResult({
+          id: 4,
+          pixScore: 62,
+          reproducibilityRate: 29.1,
+          status: AssessmentResult.status.VALIDATED,
+          emitter: 'some-emitter',
+          createdAt: new Date('2021-10-29T03:06:00Z'),
+          juryId: 100,
+          assessmentId: 2,
+          competenceMarks: [competenceMark1, competenceMark2],
+        });
+        databaseBuilder.factory.buildAssessmentResult(assessmentResult);
+        databaseBuilder.factory.buildCompetenceMark(competenceMark1);
+        databaseBuilder.factory.buildCompetenceMark(competenceMark2);
+        await databaseBuilder.commit();
+
+        // when
+        await assessmentResultRepository.updateToAcquiredLowerLevelComplementaryCertification({
+          id: 4,
+        });
+        const { commentByAutoJury } = await knex('assessment-results').select('*').where({ id: 4 }).first();
```

Comment:
Je mettrais bien une variable pour cet assessmentResultId
```suggestion
        const assessmentResultId = 4;
        const { commentByAutoJury } = await knex('assessment-results').select('*').where({ id: assessmentResultId }).first();
```

à remplacer partout 🔧 
Qu'en pensez-vous ?

Reply:
Et si je récupère plutôt l'`id` de l'`assessmentResult` créer par le domaine builder ?

J'utiliserai `assessmentResult.id` partout  🤷‍♂️ ?

Reply:
Si tu veux oui :)